mon-pix/app/services/base-url.js

```diff
@@ -0,0 +1,131 @@
+// This file is a copy of an original file from mon-pix.
+// If you need a change, modify the original file and
+// propagate the changes in the copies in all the fronts.
+
+import Service, { service } from '@ember/service';
+import ENV from 'mon-pix/config/environment';
+
+const { DEFAULT_LOCALE } = ENV.APP;
+
+// Don't use directly this service, use a url service which extends it
+export default class BaseUrlService extends Service {
+  @service currentDomain;
+  @service locale;
+
+  pixAppUrlWithoutExtension = ENV.APP.PIX_APP_URL_WITHOUT_EXTENSION;
```

Comment:
Ne serait-il pas intéressant de manipuler partout un attribut `pixAppUrl` défini une fois pour toute comme ci-dessous, plutôt que d’ajouter des `if (!this.pixAppUrlWithoutExtension) return null;` ?

```suggestion
  pixAppUrl = ENV.APP.PIX_APP_URL_WITHOUT_EXTENSION ? `${this.pixAppUrlWithoutExtension}${tld}` : ENV.rootURL;
```

On pourrait ainsi avoir des fonctions qui retourneraient des vraies valeurs non-`null` pour tous le front apps, y compris Pix App. Par exemple : 
```javascript
get pixAppCampaignRootUrl() {
    return `${this.pixAppUrl}/campagne`;
}
```